packages/gcf-utils/src/bin/genkey.ts

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env node
+/**
+ * Copyright 2019 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import fs from 'fs';
+import path from 'path';
+import commander from 'commander';
+import * as KMS from '@google-cloud/kms';
+import { Storage, StorageOptions } from '@google-cloud/storage';
+import { Base64 } from 'js-base64';
+import { Options } from 'probot';
+import * as tmp from 'tmp';
+
+commander
```

Comment:
I believe in developers making their own design decisions ... but since I wrote yargs, there's a pretty good chance I eventually swap it in for commander :stuck_out_tongue:

Reply:
WHOOPS! 😅 Maybe I'll refactor tomorrow morning

Reply:
@orthros I mean, one less repo to generate bug reports for me from; don't feel you have to.

Reply:
Too late!